app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    _, start_file_name, destination_path = command.split(" ")
+    destination_path_list = destination_path.split("/")
```

Comment:
you should use `os.path` methods to correctly split on all systems.
Same with join method below